extensions/guacamole-auth-wol/src/main/resources/templates/wol.html

```diff
@@ -0,0 +1,11 @@
+<meta name="after" content="a.home-connection" />
+<span class="wol"
+      ng-controller="wolController"
```

Comment:
It looks like you're essentially implementing a directive. It would be better to implement such a directive and reference that here, rather than use `ng-controller`.